sdk/textanalytics/ai-text-analytics/recordings/node/api_key_textanalyticsclient/recording_recognizelinkedentities.js

```diff
@@ -14,13 +14,13 @@ nock('https://endpoint', {"encodedQueryParams":true})
   'csp-billing-usage',
   'CognitiveServices.TextAnalytics.BatchScoring=1',
   'x-envoy-upstream-service-time',
-  '16',
+  '18',
```

Comment:
curious why all these recordings changed?

Reply:
I rerun the recorder to get the new service behavior.